### PR TITLE
whatsnew 2.0

### PR DESCRIPTION
**Pull request by mmerickel**
https://mmerickel-pyramid-fork.readthedocs.io/en/prep-2.0/whatsnew-2.0.html

### CODE REVIEW

File: src/pyramid/config/security.py [hunk 1]
```diff
@@ -60,8 +60,8 @@ def set_authentication_policy(self, policy):
         """
         .. deprecated:: 2.0
 
-            Authentication policies have been replaced by
-            security policies.  See :ref:`upgrading_auth` for more information.
+            Authentication policies have been replaced by security policies.
+            See :ref:`upgrading_auth_20` for more information.
```
Comment (stevepiercy) [inline src/pyramid/config/security.py]:
```suggestion
            See :ref:`upgrading_auth` for more information.
```

There is no target `upgrading_auth_20`, but `upgrading_auth` is used. Perhaps we should be explicit about the version number throughout both the source and docs and append `_20`?

**Comment by mmerickel** [comment]
Ok I added a link above to the rendered docs with a lot of changes to the docs on upgrading (ping @luhn @merwok). This is ready for review. I plan to do an alpha tomorrow and then another release once @luhn and I sort out the cookiecutter test fixtures and get that merged.

File: docs/glossary.rst [hunk 2]
```diff
@@ -87,23 +83,19 @@ Glossary
...
-     contains a ``setup.py`` file and one or more Python packages.  The
-     ``setup.py`` file contains code that allows the package(s) to be
-     installed, distributed, and tested.
+     Setuptools / Python packaging terminology.
+     A directory on disk which contains a ``setup.py`` and / or ``pyproject.toml`` file and one or more Python packages.
+     The project files contain metadata that allow the package(s) to be installed, distributed, and tested.
 
    distribution
-     (Setuptools/distutils terminology).  A file representing an
+     Setuptools / python packaging terminology.  A file representing an
```
Comment (stevepiercy) [inline docs/glossary.rst:91]:
```suggestion
     Setuptools / Python packaging terminology.  A file representing an
```

File: CHANGES.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ Features
 - Add support for Python 3.9.
   See https://github.com/Pylons/pyramid/issue/3622
```
Comment (stevepiercy) [inline CHANGES.rst:8]: The URL is plural for issues.

```suggestion
  See https://github.com/Pylons/pyramid/issues/3622
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A couple minor edits. Excellent work.

**Review by stevepiercy — APPROVED** [review]
The additional note is a good touch, especially for those who have not been watching the changes in the last year.

**Pull request merged**